tutorials/super_blink/super_blink.c

```diff
@@ -0,0 +1,51 @@
+#include "blink_config.h"
+
+#include <avr/interrupt.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define HW_TIMER_Hz     (30)
+#define SW_TIMER_SCALER (6)
+
+static volatile bool run_blink_led0 = true;
+static volatile bool run_blink_led1 = true;
+uint8_t counter = 0;
+
+void timer0_callback(void) {
+    run_blink_led0 = true;
+}
+
+void sw_timer_callback(void) {
+    run_blink_led1 = true;
+}
+
+void sw_timer(void) {
+    counter += 1;
+    if (counter % (HW_TIMER_Hz / SW_TIMER_SCALER) == 0) {
```

Comment:
wouldn't this simplify to `if (counter % 5) == 0`, running this every 5/30s of a second = 6 Hz?